framework/src/modules/token/stores/supported_tokens.ts

```diff
@@ -197,6 +197,9 @@ export class SupportedTokensStore extends BaseStore<SupportedTokensStoreData> {
 			return true;
 		}
 
-		return chainID[0] === this._ownChainID[0] && getMainchainID(chainID).equals(chainID);
+		return (
+			chainID[0] === this._ownChainID[0] &&
+			getMainchainID(chainID).equals(Buffer.alloc(CHAIN_ID_LENGTH, 0))
```

Comment:
Checking if the tokenID is LSK or not should be
* network ID is the same `chainID[0] === this._ownChainID[0]`
* chain ID is the mainchain ID `getMainchainID(chainID).equals(chainID)`
* tokenID is `[0,0,0,0]` which is mising here

Reply:
@shuse2 is the value you mention of  tokenID `[0,0,0,0]` supposed to be the `chain id` or `local id` as token id has 8 bytes `[0, 0, 0, 0, 0, 0, 0, 0]`?

Reply:
what i meant is local id. so it should be `[N, 0, 0, 0, 0, 0, 0, 0]`